dask_cuda/benchmarks/local_cupy.py

```diff
@@ -179,6 +197,40 @@ async def run(args):
                 )
                 print(fmt % (d1, d2, bw[0], bw[1], bw[2], total_nbytes[(d1, d2)]))
 
+            if args.benchmark_json:
+                import json
```

Comment:
Moving `import json` to the top should be fine

Reply:
Done! Also going to change this to `from json import dump`.